packages/zoe/src/contractFacet/seat.js

```diff
@@ -22,7 +22,8 @@ export const makeZcfSeatAdminKit = (
   let currentAllocation = harden(seatData.initialAllocation);
   let exited = false; // seat is "active"
 
-  const assertExitedFalse = () => assert(!exited, `seat has been exited`);
+  const assertExitedFalse = () =>
+    assert(!exited, details`seat has been exited`);
```

Comment:
why use details?

Reply:
Without it I got a complaint about optDetails not being a function.  Don't our `assert`s take a details object, in general?

Reply:
huh, I think you're right, thanks for catching